README.md

```diff
@@ -35,15 +35,17 @@ export default App;
 the demo requires the [React Native CLI](https://facebook.github.io/react-native/docs/getting-started.html).
 
 ```sh
-$ npm install -g react-native-cli # install victory-native if you haven't already
+$ npm install -g react-native-cli # if you haven't already
 $ git clone https://github.com/FormidableLabs/victory-native
 $ cd victory-native
-$ npm install # install victory-native
-$ cd demo
-$ npm install # install the demo's dependencies, like react-native
-$ react-native run-ios # or react-native run-android
+$ npm install   # install victory-native
+$ npm start     # start react-native packager manually before launching demo!
```

Comment:
This doesn't yield the terminal. We need clear instructions to run the rest of the commands in a _different_ terminal.